monero-outreach-docs/translations/fr/merchants/how_to_accept_monero_fr.md

```diff
@@ -0,0 +1,85 @@
+
+ La première est la plus simple: prenez-en note. Si vous faites affaire à un bas volume de transactions, vous pouvez ajouter le numéro de facture ou de transaction comme une note dans votre portefeuille lorsque vous recevez les fonds. Vous pourriez aussi prendre note du numéro d'identifiant (ID) de transaction Monero généré par votre terminal de point de vente ou par votre système comptable.
+
+ Toutefois, si vous devez absolument lier le paiement d'un client à une transaction spécifique, l'utilisation des sous-adresses est la solution à privilégier. Une sous-adresse est une adresse à usage unique, et puisqu'elles sont gratuites et illimitées, vous pouvez générer une sous-adresse pour chaque client afin de suivre chaque paiement. Cette fonctionnalité est également utile pour éviter de divulguer votre adresse principale.
+
+###_Transactions Multisig_
+
+Les transactions multisig requièrent une signature de plusieurs parties avant d'être annoncée au réseau. Par exemple, vous pourriez préparer une transaction où les fonds ne seraient relâchés jusqu'à ce que l'acheteur et le vendeur donnent conjointement leur accord. Sans explorer les détails techniques des transactions multisig ici, vous pouvez trouver plus de renseignements au lien suivant: [getmonero.org/resources/user-guides/multisig-messaging-system.html](https://www.getmonero.org/resources/user-guides/multisig-messaging-system.html)
+
+### _Entreprises LeMonero, LLC_
+
+Forts de ces détails sur le fonctionnement des transactions Monero, examinons à présent un contexte d'utilisation réel par l'entremise d'une entreprise fictive, Entreprises LeMonero LLC. Histoire inspirante d'adoption et d'intégration de Monero, Entreprises LeMonero LLC vend de la limonade et offre à ses clients de payer avec des Monero de trois manières différentes.
+
+Dans tous ces exemples, la limonade est vendue pour 1$ en monnaie fiduciaire locale. La comptabilité, ainsi que la taxation, sont également en monnaie locale. Vos conditions spécifiques varieront selon votre localisation et la règlementation entourant votre secteur d'activité.
+
+### _Kiosque LeMonero_
+
+Notre kiosque fictif possède une caisse et un carnet pour enregistrer les ventes, comme c'est souvent le cas dans les festivals ou les marchés éphémères où l'infrastructure de paiement est limitée. Dans un contexte comme celui-ci, tout ce qu'il vous faut pour accepter un paiement en Monero est un téléphone intelligent.
+
+Pour les paiements en Monero, la transaction débute de la même manière qu'avec des monnaies fiduciaires: vous préparez la limonade, calculez le montant dû et enregistrez la transaction dans votre carnet (selon le format qui convient à vos besoins de comptabilité).
+
+Le montant de 1$ est ensuite converti en XMR avant d'initier le paiement. Le taux de change est plutôt variable: au moment d'écrire ces lignes, 1 XMR coûte $146.35 USD, donc une limonade à 1$ coûterait 0.0068 XMR.
```

Comment:
1 XMR coûte $146.35 USD, donc une limonade à 1$ coûterait 0.0068 XMR.  

Comment: In the translation, values were changed I presume compared to the original. It is not critical, but is this really necessary? In particular XMR is closer to 220 USD at the time I review this.

Reply:
Makes sense. Changed to 278.34$!